lib/gamora/client.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+module Gamora
+  class Client < OAuth2::Client
+    def self.from_config
+      new(
+        Configuration.client_id,
```

Comment:
I find more easy to main to have a config object with the config properties set, and use that instead of module attributes as in this case,

Reply:
That how the `Client` works. Configuration is more like an static and general data. If we want to have a `Client` with diferent values can be done in its initializer.